com.unity.cinemachine/Samples~/3D Samples/Clearshot/AnimateBasedOnCartSpeed.cs

```diff
@@ -0,0 +1,47 @@
+using Cinemachine;
+using UnityEngine;
+using UnityEngine.Splines;
+
+/// <summary>
+/// Propagates the spline carts speed to the animator. If no spline cart is found, it calculates a position delta.
+/// </summary>
+[RequireComponent(typeof(Animator))]
```

Comment:
I'm working on this idea, so don't spend too much time on this.  I noticed that the animation in Cameron was broken, because of the slowed-down idle.  The whole controller needs to be redone.

Reply:
yeah, no longer need a custom script for this.
Yay! one less.